Packs/ServiceNow/Integrations/ServiceNow/ServiceNow.py

```diff
@@ -1377,6 +1386,137 @@ def get_table_name_command():
     return entry
 
 
+def query_items():
+    args = unicode_to_str_recur(demisto.args())
+    table_name = 'sc_cat_item'
+    category = args.get('item-category')
+    offset = args.get('offset', DEFAULTS['offset'])
+    limit = args.get('limit', DEFAULTS['limit'])
+
+    items_query = ''
+
+    if category:
+        items_query = 'category.title=' + category
+
+    res = query(table_name, limit, offset, items_query)
+
+    if not res or 'result' not in res:
+        return 'No items found'
+
+    items = res['result']
+    if not isinstance(items, list):
+        items = [items]
+
+    if len(items) == 0:
+        return 'No items found'
+
+    mapped_items = []
+    for item in items:
+        mapped_items.append(get_item_human_readable(item))
+
+    entry = {
+        'Type': entryTypes['note'],
+        'Contents': res,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('ServiceNow Catalog Items', mapped_items,
+                                         removeNull=True, headerTransform=pascalToSpace),
+        'EntryContext': {
+            'ServiceNow.CatalogItem(val.ID===obj.ID)': createContext(mapped_items, removeNull=True),
+        }
+    }
+    return entry
+
+
+def get_item_command():
+    args = unicode_to_str_recur(demisto.args())
+    sys_id = args.get('sys-id')
+    res = send_request('/sn_sc/servicecatalog/items/' + sys_id, 'get')
+
+    if not res or 'result' not in res:
+        return 'Item not found'
+
+    res = res['result']
+
+    mapped_item = get_item_human_readable(res)
+
+    entry = {
+        'Type': entryTypes['note'],
+        'Contents': res,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('ServiceNow Catalog Item', mapped_item,
+                                         removeNull=True, headerTransform=pascalToSpace),
+        'EntryContext': {
+            'ServiceNow.CatalogItem(val.ID===obj.ID)': createContext(mapped_item, removeNull=True),
+        }
+    }
+    return entry
+
+
+def order_item_command():
+    args = unicode_to_str_recur(demisto.args())
+    sys_id = args.get('sys-id')
+    quantity = args.get('quantity', '1')
+    variables = args.get('variables', '')
```

Comment:
Where are they getting the variables from? I didn't see them in any output.

Reply:
the variables values comes from the table variables inside category item(sc_cat_item), each item can have variables, The item Paper and Supplies for example:

![image](https://user-images.githubusercontent.com/46249224/79851562-29711580-83ce-11ea-8d7a-46a3d0b42821.png)

Reply:
According to this in the issue:
`Get the item variables using the /sn_sc/servicecatalog/items/{sys_id} endpoint.`
They need the variables from the item details command. It should be in the output.

Reply:
no problem, i'll add it